config/config.json

```diff
@@ -145,7 +145,9 @@
         "UsernameAttribute": "",
         "IdAttribute": "",
         "SkipCertificateVerification": false,
-        "QueryTimeout": 60
+        "QueryTimeout": 60,
+        "LoginFieldName": "",
```

Comment:
Maybe we should have a branding section? It seems like configuring LDAP and customizing the look of the software are two different tasks. Although I might be thinking about this setting while I am configuring LDAP.  0/5